.github/workflows/build.yml

```diff
@@ -196,7 +196,7 @@ jobs:
           # This is a workaround for a SSL false positive in cmake 3.26.4
           # When downloading the manual. 3.21 is required for installing the
           # ANGLE Dlls via IMPORTED_RUNTIME_ARTIFACTS
```

Comment:
Can you update/extend the comment? Even Ubuntu Jammy (min for Mixxx 2.5)  has cmake 3.22.